modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/upload/DDMFormUploadResponseHandler.java

```diff
@@ -59,6 +60,10 @@ public JSONObject onFailure(
 				StringUtil.merge(
 					_ddmFormUploadValidator.getGuestUploadFileExtensions()));
 		}
+		else if (portalException instanceof FileNameException) {
+			errorMessage = themeDisplay.translate(
+				"please-enter-a-file-with-a-valid-file-name");
```

Comment:
@carolmariaabb, what is a valid file name? I think the customer can be in doubt about it.

Is there any chance of we increment this message to make it more specific, like `Please enter a file with a valid file name (filename.doc)`?

What do you think?

Maybe, Russ can help us. 🙂

Reply:
I will talk to Russell 😄 I'll let you know when I have an answer.